content/advent-2016/exposing-go-on-the-internet.md

```diff
@@ -0,0 +1,212 @@
++++
+date = "2016-12-19T01:30:27Z"
+author = ["Filippo Valsorda"]
+title = "So you want to expose Go on the Internet"
+linktitle = "Exposing Go on the Internet"
+series = ["Advent 2016"]
++++
+
+Back when `crypto/tls` was slow and `net/http` young, the general wisdom was to always put Go servers behind a reverse proxy like Nginx. That's not necessary anymore!
+
+However, the Internet is the deep end of the pool when it comes to networks, and there are a few things you have to do to teach your server to swim before throwing it in.
+
+At [Cloudflare][cf] we recently experimented with exposing pure Go services to the hostile wide area network. Here are some hard-learned lessons.
+
```

Comment:
This would be a great spot to put the positive spin on:
`The net/http package is fast, complete, and awesome, but the defaults might not be great for a production server.  In this article I'll show you how to go from great to Battle Tested hardening for your net/http server.`  
Use your words of course, not mine.